packages/sdk/test/integration/gc_test.ts

```
@@ -383,21 +670,45 @@
       root.list = [0, 1, 2];
       root.list.push([3, 4, 5]);
     });
+    assert.equal(
+      versionVectorHelper(doc.getVersionVector(), [
+        { actor: cli.getID()!, lamport: BigInt(2) },
+      ]),
+      true,
+    );
     assert.equal('{"list":[0,1,2,[3,4,5]]}', doc.toJSON());
     doc.update((root) => {
       delete root.list[1];
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)